invenio_app_ils/ui/src/semantic-ui/site/elements/container.overrides

```diff
@@ -20,80 +35,137 @@
     }
 
     &.fs-content {
-    margin-top: @headerMenuHeight;
-    flex: 1;
-    justify-content: space-between;
-  }
+      margin-top: @headerMenuHeight;
+      flex: 1;
+      justify-content: space-between;
+    }
 
     &.document-tags {
       margin-top: 2em;
       margin-bottom: 2em;
     }
-  }
 
-  .breadcrumbs{
-    padding-bottom: 2em;
-  }
+    &.fs-headline-section {
+      background-color: rgba(0, 0, 0, .3);
+      background-image: @headlineBackground;
+      min-height: 75vh;
+      background-size: cover;
+      background-position: center;
+      background-blend-mode: darken;
+    }
 
-  /* Frontsite search bar container */
+    &.fs-headline {
+      padding: @defaultPaddingValue;
+      color: white;
 
-  .ui.container.document-details-search-container {
-    background-color: @highlightBackgroundColor;
-    padding-top: 2em;
-    padding-bottom: 2em;
-    border-bottom: 1px solid #c6c4d0;
-    position: relative;
-  }
+      .container-header {
+        margin-bottom: 2em;
+        margin-top: 6em;
+      }
 
-  .ui.container.document-details-search-container:before,
-  .ui.container.document-details-search-container:after {
-    content: "";
-    position: absolute;
-    z-index: -1;
-    box-shadow: 0 0 20px -8px rgba(0, 0, 0, 0.2);
-    top: 50%;
-    bottom: 0;
-    left: 43%;
-    right: 43%;
-    border-radius: 100px / 10px;
-  }
+      .container-extra {
+        margin-top: 7em;
+      }
+    }
 
-  /* frontsite footer */
+    /* Wrap the dot background with this container */
 
-  .ui.container.footer-lower {
-    background-color: #2A2D34;
-    padding-top: @footerPaddings/2;
-    padding-bottom: @footerPaddings/2;
-    color: white;
-  }
+    &.dot-background-container {
+      position: relative;
+    }
 
-  .ui.container.footer-upper {
-    background: rgb(52, 55, 62);
-    background: linear-gradient(155deg, rgba(52, 55, 62, 1) 0%, rgba(71, 75, 85, 1) 100%);
-    padding-top: @footerPaddings;
-    padding-bottom: @footerPaddings;
-    color: #c1bbbb;
-  }
+    &.dot-background {
+      &:after {
+        content: "";
+        background: linear-gradient(90deg, white (@dot-bg-space - @dot-bg-size) 50%, transparent 1%) center,
+        linear-gradient(white (@dot-bg-space - @dot-bg-size), transparent 1%) center,
```

Comment:
If the gradients are the same it can be a reusable variable.

Reply:
they are not, but I will separate the whole element if it will make sense somewhere